frontend/src/components/WofEditPopup.vue

```diff
@@ -0,0 +1,123 @@
+<!-- 
+Component changes something.
+Example:
+<wof-edit-popup title="Edit username" :visible="true" currentValueName="Username" currentValue="wofUser123"/>
+-->
+
+
+<template>
+    <wof-info-box class="wof-edit-popup" :title="this.title" :isOpen="this.state" v-if="this.state"  @close="changeState">
+
+        <div class="wof-edit-popup__field">
+            <form @submit.prevent="submitForm">
+                <wof-input :name="this.currentValueName" :v-model="this.value" :initValue="this.currentValue" @change="editValue"  :error="this.errorMsg" ></wof-input>
+            </form>
+        </div>
+
+        <div class="wof-edit-popup__buttons">
+            <wof-button variant="outline" @click="changeState">Cancel</wof-button>
+            <wof-button variant="positive" @click="submitForm">Submit</wof-button>
+        </div>
+
+    </wof-info-box>
+
+    <!--Not used at this moment-->
+    <div v-if="loading" class="loading">
+        <wof-spinner-dots :size="4" :loading="loading"></wof-spinner-dots>
+    </div>
+
+
+
+</template>
+
+<script>
+import WofInfoBox from '../components/WofInfoBox.vue';
+import WofInput from '..//components/WofInput.vue';
+import WofSpinnerDots from "../components/WofSpinnerDots.vue";
+export default {
+    name: 'WofEditPopup',
+    components: { WofInfoBox, WofInput, WofSpinnerDots },
+    props:{
+        title:{
+            type: String,
+            default: ""
+        },
+        visible: {
+            type: Boolean,
+            default: false
+        },
+        currentValueName: {
+            type: String,
+            default: "Something"
+        },
+        currentValue: {
+            type: String,
+            default: ''
+        }
+    },
+    data(){
+        return{
+            state: this.visible,
```

Comment:
Move it to props and change name to 'isOpen'.
We will open this popup from outside.